packages/e2e-tests/specs/editor/various/rtl.test.js

```diff
@@ -21,6 +21,37 @@ describe( 'RTL', () => {
 		} );
 	} );
 
+	// Regression test for https://github.com/WordPress/gutenberg/issues/29250.
+	it( 'loads editor without errors when RTL is set at the WP admin level', async () => {
+		await page.evaluate( () => {
+			window._loadedWithoutErrors = true;
+
+			window.addEventListener( 'error', function ( event ) {
+				window._loadedWithoutErrors = false;
+				event.preventDefault();
+			} );
+
+			const scripts = Array.from( document.querySelectorAll( 'script' ) );
+			const snippetScriptEl = document.createElement( 'script' );
+			const newComponentsScriptEl = document.createElement( 'script' );
+			const head = document.head;
+
+			const componentsScriptSrc = scripts.find( ( s ) =>
+				s.src.match( 'components/index.js' )
```

Comment:
I don't quite like the fact that this ends up coupling this test with the build output (to a portion of the path to the compiled entry-point for `@wordpress/components`), but maybe it's fine as it won't change often ¯\_(ツ)_/¯

The alternative would be to just get all scripts without filtering, unshift the `snippetScriptEl`, and then re-add all of them again later. I tried that approach and it works, but then we would be re-evaluating all scripts, which is quite inefficient because we only really need to reload `@wordpress/components`. 

As much as I don't quite like the current approach, I think it provides the best trade-offs. Both approaches would be kind of hacky, but I don't see any other way of reproducing the crash caused by `rtlcss` from an e2e test. See my other attempts [here](https://github.com/WordPress/gutenberg/pull/29326#issuecomment-786139476).

Reply:
Another thing is that there might be a possibility of a false-positive if the error is caught by the global error handler after the test finishes, which would mean the `window._loadedWithoutErrors` would still be `true`. While testing locally, the test consistently failed when I tested against `v10.0.0` though, but I think it's a possibility :/. I didn't find a good way to wait for the error without it creating a bottleneck in the case the error doesn't happen.